packages/tocco-ui/src/Menu/ButtonMenu.js

```diff
@@ -63,23 +64,27 @@ ButtonMenu.propTypes = {
   /**
    * Optional handler. If defined, a split button will be rendered
    */
-  onClick: PropTypes.func,
+  'onClick': PropTypes.func,
```

Comment:
(quotes inserted intentionally in the proptypes here?)

Reply:
thats the linting rule. if one property names requires quotes, every attribute should have quotes. Autofix did this